pyanaconda/ui/common.py

```diff
@@ -665,7 +665,7 @@ def collectCategoriesAndSpokes(paths, klass):
     ret = {}
     # Collect all the categories this hub displays, then collect all the
     # spokes belonging to all those categories.
-    categories = sorted(collect_categories(paths["categories"]), key=lambda c: c.sortOrder)
+    categories = sorted(collect_categories(paths["categories"]), key=lambda c: c.get_sort_order())
```

Comment:
This sort doesn't seem to be useful.

Reply:
I assumed there is some reason we sort the categories when they are being collected (stable listing in logs ?) so I just kept the logic the same. If you think it's safe to drop this, I can definitely do that. :)

Reply:
I would try to remove drop that. It is just confusing.